app/models/card.rb

```diff
@@ -16,4 +17,20 @@ def check_duplication
   def set_default_date
     self.review_date = Date.today + 3.days
   end
+
+  def check_translations(submitted_text)
+    original = card_downcase(self.original_text)
+    submitted = card_downcase(submitted_text)
+    if original == submitted
+      self.update_attribute(:review_date, (Date.today + 3.days))
+      return true
+    else
+      return false
+    end
+  end
+
+  def card_downcase(string)
```

Comment:
Unused method argument - `string`. If it's necessary, use `_` or `_string` as an argument name to indicate that it won't be used. You can also write as `card_downcase(*)` if you want the method to accept any arguments but don't care about them.